recipes/tbb/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# clang: error: invalid deployment target for -stdlib=libc++
+export MACOSX_DEPLOYMENT_TARGET=10.7
+
+make -j${CPU_COUNT}
```

Comment:
Please drop `CPU_COUNT`. CIs are bad about providing an accurate example. For instance, they sometimes report the cores on the node instead of how many we are allotted.